src/FrontDoor/FrontDoor/Cmdlets/NewFrontDoorWafPolicy.cs

```diff
@@ -142,10 +168,23 @@ public override void ExecuteCmdlet()
                     CustomBlockResponseBody = CustomBlockResponseBody == null ? CustomBlockResponseBody : Convert.ToBase64String(Encoding.UTF8.GetBytes(CustomBlockResponseBody)),
                     CustomBlockResponseStatusCode = this.IsParameterBound(c => c.CustomBlockResponseStatusCode) ? CustomBlockResponseStatusCode : (int?)null,
                     RedirectUrl = RedirectUrl,
-                    RequestBodyCheck = this.IsParameterBound(c => c.RequestBodyCheck) ? RequestBodyCheck : PSEnabledState.Enabled.ToString()
+                    RequestBodyCheck = this.IsParameterBound(c => c.RequestBodyCheck) ? RequestBodyCheck : PSEnabledState.Enabled.ToString(),
                 },
-                Sku = this.IsParameterBound(c => c.Sku) ? new Management.FrontDoor.Models.Sku(Sku) : null
+                Sku = this.IsParameterBound(c => c.Sku) ? new Management.FrontDoor.Models.Sku(Sku) : null,
             };
+
+            if (JavascriptChallengeExpirationInMinutes >=5 && JavascriptChallengeExpirationInMinutes <= 1440)
```

Comment:
As I said in other comment, this is not mandatory for some SKUs, let us remove the validation here and let the service to conduct the validation.